_infra/helm/securebanking-test-data-initializer/values.yaml

```diff
@@ -1,3 +1,4 @@
+deploymentType: CronJob
```

Comment:
This is the chart that customers will pick up? If so, then I think Job is a better default which we can then override for dev.

The values file also contains stuff that is very much FR specific, would be good to get this cleaned up. If it is too messy to change now then a separate issue will suffice.